rfcs/template.md

```diff
@@ -0,0 +1,68 @@
+# Descriptive Name for the Proposal
+
+## Introduction
+
+Short description of the idea proposed with explained motivation. 
+
+The motivation could be:
+- Improved users experience for API changes and extensions. Code snippets to
+  showcase the benefits would be nice here.
+- Performance improvements with the data, if available.
+- Improved engineering practices.
+
+Introduction may also include any additional information that sheds light on
+the proposal, such as history of the matter, links to relevant issues and
+discussions, etc.
```

Comment:
Would it make sense to format the boilerplate text int the template to be italicized and maybe put into some brackets?

Reply:
Looking at the rendered text, I think it's clear without brackets. But we can revisit later if people are confused.